recipes/flatbuffers/all/conanfile.py

```diff
@@ -18,13 +18,11 @@ class FlatbuffersConan(ConanFile):
                "fPIC": [True, False],
                "header_only": [True, False],
                "flatc": [True, False],
-               "flatbuffers": [True, False],
-               "options_from_context": [True, False]}
```

Comment:
This option should be deprecated, not removed so fast. Removing an option can break downstream projects.

Reply:
I can leave the option there and print a message if the value changes from the default... but the behavior introduced by this option is something we need to remove.

Here in CCI only `gdal` uses this recipe and it should work the same (yes, there is the bug related to package-id, but we can regenerate `gdal` if we merge these changes).

Reply:
Yes, the idea is to:
- add the value "deprecated" in possible values
- change default value to "deprecated"
- display a warning in configure if value is different than "deprecated"
- delete option in package_id
- remove any other logic of this option in the recipe